src/main/java/com/stakater/kubernetes/assertions/NavigationListAssert.java

```diff
@@ -1,7 +1,6 @@
 package com.stakater.kubernetes.assertions;
 
-import org.assertj.core.api.AbstractAssert;
-import org.assertj.core.api.ListAssert;
+import org.assertj.core.api.*;
```

Comment:
never use wildcard imports; disable settings inside IntelliJ

Reply:
I didn't know intelliJ was doing it, i'll disable it and add specific imports